src/NuGet/VisualStudio/VS.ExternalAPIs.Roslyn.Package.csproj

```diff
@@ -44,6 +44,7 @@
     <ProjectReference Include="..\..\VisualStudio\Core\Def\Microsoft.VisualStudio.LanguageServices.csproj" PrivateAssets="all"/>
     <ProjectReference Include="..\..\VisualStudio\Core\Impl\Microsoft.VisualStudio.LanguageServices.Implementation.csproj" PrivateAssets="all"/>
     <ProjectReference Include="..\..\VisualStudio\Core\SolutionExplorerShim\Microsoft.VisualStudio.LanguageServices.SolutionExplorer.csproj" PrivateAssets="all"/>
+    <ProjectReference Include="..\..\VisualStudio\Core\SolutionExplorerShim\Microsoft.VisualStudio.LanguageServices.SolutionExplorer.csproj" PrivateAssets="all"/>
```

Comment:
@jasonmalinowski Intentional duplicate reference?

Reply:
@sharwell No, that wasn't. Did you fix this one up?

Reply:
> No, that wasn't. Did you fix this one up?

I did not. Once I had a passing build I didn't want to chance the build queue ignoring my manual override and starting a new cycle of failures.